src/types.rs

```diff
@@ -177,3 +177,130 @@ impl<'a> From<&'a QUIC_CONNECTION_EVENT> for ConnectionEvent<'a> {
         }
     }
 }
+
+/// Stream callback events
+pub enum StreamEvent<'a> {
+    StartComplete {
+        status: crate::Status,
+        id: crate::u62,
+        peer_accepted: bool,
+    },
+    Receive {
+        absolute_offset: u64,
+        total_buffer_length: &'a mut u64,       // out parameter
```

Comment:
nit: "inout" parameter would be more accurate, since it is also indicating the total number of bytes indicated by the event.

Reply:
done.